src/index.html

```diff
@@ -13,20 +13,36 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="styles/style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <section
```

Comment:
Use the semantic tag for form
```suggestion
    <form
```